src/libcdio/cdio/ecma_167.h

```diff
@@ -334,6 +334,7 @@ struct anchor_vol_desc_ptr_s
 } GNUC_PACKED;
 
 typedef struct anchor_vol_desc_ptr_s anchor_vol_desc_ptr_t;
+typedef struct anchor_vol_desc_ptr_s *ptr_anchor_vol_desc_ptr_t;
```

Comment:
I'm afraid I will not patch any of the src/libcdio/** files unless they are patched in the upstream project (https://www.gnu.org/software/libcdio/), so, if you will need to send this part of your patch (based on the actual libcdio code and not the Rufus version) to the libcdio-devel mailing list at https://savannah.gnu.org/mail/?group=libcdio.